ui/packages/components/src/RunsPage/columns.tsx

```diff
@@ -33,6 +56,52 @@ const columns = [
     },
     header: 'Run ID',
     enableSorting: false,
+    id: ensureColumnID('id'),
+  }),
+  columnHelper.display({
+    cell: (props) => {
+      const data = props.row.original;
+
+      if (data.isBatch) {
+        return 'Batch';
+      }
+      if (data.cronSchedule) {
+        return data.cronSchedule;
```

Comment:
**design**: Design wise, we should make at least crons monospace. We can use IDCell for now, but ideally that should be more a more generic component name.